### ISSUE TITLE
Food Check showing when not in whitelisted area

### ISSUE THREAD

**Issue by grizzilk-codes** [MidoriKami/NoTankYou#20]
This is the same issue as https://github.com/MidoriKami/NoTankYou/issues/18 that was closed without enough info.

When Banner Settings is set to Solo Mode - Everywhere but Food Check is set to only Whitelisted Areas, it appears the whitelisted areas are not enforced.

```
Currently Whitelisted Areas:

{1003,1005,1004,1002,1007,1006,1008,1009}
```
```
"Currently in MapID: [397]"
```

I attempted to add the zone and then remove it but the banner for Food Check persists. The only way to remove it is to either eat food, turn the food check alert off all together, or turn on Party Mode.

**Comment by grizzilk-codes** [comment]
My C is rudimentary at best, but I think what is happening is:

When in solo mode, https://github.com/MidoriKami/NoTankYou/blob/9ed6e6ab46556520405d169b7fd39f18fcd46617/NoTankYou/DisplaySystem/Banners/FoodBanner.cs#L21 this code is returning after determining solo mode is enabled and is either never reaching the code that checks for whitelisted area: https://github.com/MidoriKami/NoTankYou/blob/9ed6e6ab46556520405d169b7fd39f18fcd46617/NoTankYou/DisplaySystem/Banners/FoodBanner.cs#L29 or the whitelisted area check doesn't exist in UpdateSolo: https://github.com/MidoriKami/NoTankYou/blob/9ed6e6ab46556520405d169b7fd39f18fcd46617/NoTankYou/DisplaySystem/Banners/FoodBanner.cs#L47

**Comment by MidoriKami** [comment]
Solo mode intentionally ignores whitelists.

**Comment by grizzilk-codes** [comment]
There is a check box inside the Food Check Warnings page that indicates "Food Solo Mode" is disabled. This modifies the screen to display a Whitelist. Is this ignored if global solo mode is turned on? If so, is there a way around this? Where someone could run the other alerts in Solo mode(faerie, carbuncle, etc), while not being told about food?

**Comment by MidoriKami** [comment]
No as that is currently outside the intended use cases for the plugin.

Describe why you feel this feature is necessary.

**Comment by grizzilk-codes** [comment]
My specific use case is for Eureka, I don't want to job switch and forget that I switched to a SMN or SCH until a battle starts and realize I'm missing my pet, as I tend to do. There's no real benefit to eating food, so I don't need the warnings to tell me about it. However, I would like the food warning to let me know if I go into p[1-4]{s}. The savage content would be grouped, so solo mode shouldn't be necessary. However, having solo mode for the other checks would be helpful.

The more I type this out, I realize the actual feature request might be: "Can I set the entire plugin to group mode, but make it so Carbuncle and Faerie can be set to solo mode?"
